index.html

```diff
@@ -0,0 +1,195 @@
+
+      <h2>Usage</h2>
+
+      <pre class="snippet" lang="no-highlight" style="padding: 0">
+        <button class="btn" onclick="copyToClipboard(this)">
+          <img class="clippy" src="images/clippy.svg" alt="Copy to clipboard" width="13">
+        </button>
+  <code id="helm-command">helm repo add terascope https://terascope.github.io/helm-charts</code>
+      </pre>
+
+      <script>
+        function copyToClipboard(button) {
+          
+          const codeBlock = button.nextElementSibling;
+          const textToCopy = codeBlock.textContent;
+
+          
+          navigator.clipboard.writeText(textToCopy).then(() => {
+            
+            
+          }).catch((err) => {
+            
+          });
+        }
+      </script>
+
+      <p>Run the command above to add the repo to the helm cli client. This will contain all the charts listed under the <a href="#chart-1">Charts section</a> below. This can be validated by running the helm search command below:</p>
+      <pre class="snippet" lang="no-highlight" style="padding: 0">
+        <button class="btn" onclick="copyToClipboard(this)">
+          <img class="clippy" src="images/clippy.svg" alt="Copy to clipboard" width="13">
+        </button>
+  <code id="helm-search-command">helm search repo terascope --versions</code>
+      </pre>
+
+      <h2>Reference Links</h2>
+      <p>
+        <ul>
+          <li><a href="https://github.com/terascope/helm-charts">Terascope/helm-charts GitHub</a></li>
+        </ul>
+      </p>
+
+      <h2 id="chart-1">Charts</h2>
+
+      <div class="charts">
+			
+        
+          <div class="chart">
+            <a href="https://github.com/terascope/helm-charts/releases/download/kafka-1.2.0/kafka-1.2.0.tgz" title="https://github.com/terascope/helm-charts/releases/download/kafka-1.2.0/kafka-1.2.0.tgz">
+              <div class="icon">
+                <img class="chart-item-logo" alt="kafka's logo" src="https://kafka.apache.org/images/apache-kafka.png" onerror="this.onerror=null; this.src='images/placeholder.png';">
+              </div>
+              <div class="body">
+                <p class="info">
+                  kafka
+                  (1.2.0@3.7.1)
+                  <a href="https://github.com/terascope/helm-charts/releases/tag/kafka-1.2.0">
+                    <img src="images/GitHub-Mark-32px.png" alt="github link" style="height: 16px; width: 16px; vertical-align: middle;" />
+                  </a>
+                </p>
+                <p class="description">
+                  Apache Kafka is publish-subscribe messaging rethought as a distributed commit log.
+                </p>
+              </div>
+            </a>
+          </div>
+        
+			
+      </div>
+    </section>
+		<time datetime="2025-03-18T23:54:08" pubdate id="generated">Tue Mar 18 2025 11:54:08PM UTC&#43;00:00</time>
+
+    <script src="https://unpkg.com/clipboard@2/dist/clipboard.min.js"></script>
```

Comment:
I will point out again a place where you introduce a third party service as a dependency and put them in a position of monitoring people who interact with the site you're generating.  Who is unpkg.com,  Why do you want them to see everyone who uses this webpage?  You are giving these people the ability to execute arbitrary javascript in the browser of every person who uses this page.

Reply:
Sorry, this was copied over from teraslice. I assumed it was vetted already. I'll remove it from both places.